heft-plugins/heft-webpack5-plugin/tsconfig.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "./node_modules/local-node-rig/profiles/default/tsconfig-base.json"
+  "extends": "./node_modules/local-node-rig/profiles/default/tsconfig-base.json",
+  "compilerOptions": {
+    "skipLibCheck": true
```

Comment:
Why is this needed? Also same for the other ones. In general we should try to avoid using this if we can.